packages/library/src/send-transaction.ts

```diff
@@ -55,6 +60,23 @@ type SendAndConfirmDurableNonceTransactionFunction = (
     >,
 ) => Promise<void>;
 
+type SendTransactionWithoutConfirmingFunction = (
+    transaction: SendableTransaction,
+    config: Omit<Parameters<typeof sendTransaction_INTERNAL_ONLY_DO_NOT_EXPORT>[0], 'rpc' | 'transaction'>,
+) => Promise<void>;
+
+export function sendTransactionWithoutConfirmingFactory({
```

Comment:
Without confirming? Without confirmation?

Reply:
`WithoutConfirming` makes more sense to me. `WithoutConfirmation` _could_ imply there's no confirmation that the transaction was sent.

Reply:
+1 on `WithoutConfirming`.